index.html

```diff
@@ -489,20 +489,24 @@
           <a href='https://www.w3.org/TR/ldp/#ldpc-HTTP_DELETE'>section 5.2.5</a>, it must also follow the additional
           behavior outlined below.
         </p>
-        <section id="httpDELETEDepth">
-          <h3>Depth Header</h3>
+        <section id="httpDELETERecursion">
+          <h3>Recursive Delete</h3>
           <p>
-            A Fedora server MUST support the <code>Depth</code> header, as defined in [[!RFC4918]]
-            <a href='https://tools.ietf.org/html/rfc4918#section-10.2'> section 10.2 </a>. Implementations MAY choose
-            not to support all of the header's values and MUST reject a request containing an unsupported
-            <code>Depth</code> value with a 400 (Bad Request). The default value for the <code>Depth</code> header if
-            not present in a request is left up to the implementer.
-          </p>
-          <p>
-            If a server supports recursive <code>DELETE</code>, that recursion MUST be reckoned along the [[!LDP]]
+            If a server supports <code>DELETE</code>, its recursion MUST be reckoned along the [[!LDP]]
```

Comment:
I would change "its recursion" to "any recursion", since supporting DELETE and recursing are two different concerns.